Makefile

```diff
@@ -223,13 +223,15 @@ test-ci-js: | clear-stalled
 	$(PYTHON) tools/test.py $(PARALLEL_ARGS) -p tap --logfile test.tap \
 		--mode=release --flaky-tests=$(FLAKY_TESTS) \
 		$(TEST_CI_ARGS) $(CI_JS_SUITES)
+	! ( ps awwx | grep Release/node | grep -v grep )
```

Comment:
I'm pretty sure they're all used in #11246, so if they aren't we should find out pretty quickly 😅